examples/calendar/ReadMe.md

```diff
@@ -0,0 +1,50 @@
+## How to run this project?
+
+You can use the following commands to run the code
+
+### YARN
+
+```
+yarn
```

Comment:
small thing, but you commited package-lock, so this implies npm, also on top level you have yarn.lock, i think we should be consistent here